nf_core/pipeline-template/lib/WorkflowMain.groovy

```diff
@@ -82,13 +82,12 @@ class WorkflowMain {
     //
     // Get attribute from genome config file e.g. fasta
     //
-    public static String getGenomeAttribute(params, attribute) {
-        def val = ''
+    public static Object getGenomeAttribute(params, attribute) {
         if (params.genomes && params.genome && params.genomes.containsKey(params.genome)) {
             if (params.genomes[ params.genome ].containsKey(attribute)) {
-                val = params.genomes[ params.genome ][ attribute ]
+                return params.genomes[ params.genome ][ attribute ]
             }
         }
-        return val
+        return ''
```

Comment:
I think you likely what to return `null` to be semantically similar to the previous return an empty string if attribute not found.